### PR TITLE
Add logging

### PR DESCRIPTION
**Pull request by tcnichol**
This pull request adds a logger (plus several log statements) in brapi. 

Not sure if it is correct, feedback needed.

### CODE REVIEW

**Comment by dlebauer** [comment]
This seems to have the same changes as #10 along with the new logging ... perhaps it would make more sense to have the issue-537/cultivar-treatment-siteclear branch be the base for the pr instead of off master?

File: bety_brapi/controllers_impl/StudiesController_impl.py [hunk 1]
```diff
@@ -58,14 +60,14 @@ def studies_study_db_id_get(studyDbId):
...
-    print(query)
+    logger.info(query)
 
     if studyDbId:
         query += " AND experiments.id = %s "
         # query = query % studyDbId
         params.append(studyDbId)
 
-    print(query)
+    # print(query)
```
Comment (robkooper) [inline bety_brapi/controllers_impl/StudiesController_impl.py]: use logger.debug when you want to print the query. When we run in production we will run with loglevel INFO.

**Review by robkooper — CHANGES_REQUESTED** [review]

**Comment by robkooper** [comment]
@tcnichol merging this request, but you are making changes to the logging branch that should be happening on the original branch. Please make changes from now on the issue-537/cultivar-treatment-siteclear branch

**Comment by robkooper** [comment]
Actually can't merge due to conflict, can you fix conflict and then just merge.

**Review by robkooper — APPROVED** [review]

**Comment by robkooper** [comment]
@tcnichol this seems to have more than just logging, what was else was done and why?

**Comment by robkooper** [comment]
Working on a large rewrite of code, not sure what this does and how it needs to be incorperated in rewrite.

**Comment by tcnichol** [comment]
Rob.

Some of the other stuff was that this was built off the other brapi pull request adding the new endpoints.

This just changes how logging is done, using an actual logger instead of print statements. I think it would be best that I decline this pull request, and if the feature is still wanted I can create a new one against master that would be easier to merge.

**Comment by tcnichol** [comment]
this was built off an earlier branch and many of the changes no longer make sense so i am closing this one without mergin.

**Pull request closed without merging**